### DISCUSSION TITLE
discord机器人始终无法回复消息

### DISCUSSION THREAD

**Discussion by dalamudx** [nonebot/discussions#468] [问答] [answered]
配置如下
```
HOST=0.0.0.0
PORT=8080
ONEBOT_ACCESS_TOKEN=TOKEN
DRIVER=~fastapi+~httpx+~websockets
COMMAND_START=["/"]
DISCORD_BOTS='
[
  {
    "token": "TOKEN",
    "intent": {
      "guild_messages": true,
      "direct_messages": true
    },
    "application_commands": {"*": ["*"]}
  }
]
'
```
机器人权限
![image](https://github.com/nonebot/discussions/assets/88395653/2b643d56-708c-4029-af7b-cb78e3562cf6)
![image](https://github.com/nonebot/discussions/assets/88395653/1e4278a6-ba89-4295-8edd-35d061fb0655)
日志
![image](https://github.com/nonebot/discussions/assets/88395653/a6cd9aa5-e092-442e-84e1-afac488438be)
问题
在discord频道里使用`@机器人 /echo test`日志能接收到，但是没有回应，一直以为是机器人权限问题，但即使给了管理员权限也还是一样没有回应.同时接了NTQQ插件，从QQ里却是能正常回复
版本
nonebot-adapter-console 0.6.0
nonebot-adapter-discord 0.1.7
nonebot-adapter-onebot  2.4.3
nonebot2                2.3.1

**Comment by yanyongyu** [comment]
你可以打开debug日志查看是否执行了响应器

**Reply by dalamudx** [reply]
![image](https://github.com/nonebot/discussions/assets/88395653/98652bc9-4ea0-42fc-bd46-e98ce0b68bdc)
应该是没有，日志里ONEBOT V11上面是从discord频道里发送消息之后的日志，然后下面的是通过QQ发的

**Reply by yanyongyu** [reply, marked as answer]
看上去是没有匹配上命令，这个问题你得问适配器，是不是因为没有去除掉开头at机器人的消息

**Reply by dalamudx** [reply]
> 看上去是没有匹配上命令，这个问题你得问适配器，是不是因为没有去除掉开头at机器人的消息

OK
